Makefile

```diff
@@ -4,13 +4,11 @@ PREFIX ?= /
 
 .PHONY: install
 install:
-	install -D -m0644 src/overdue.awk $(PREFIX)/usr/share/overdue/overdue.awk
 	install -D -m0644 src/overdue-libraries-upgrade.hook \
 		$(PREFIX)/usr/share/libalpm/hooks/overdue-libraries-upgrade.hook
-	install -D -m0755 src/overdue.sh  $(PREFIX)/usr/bin/overdue
```

Comment:
I don't know the answer to this one, but I liked being able to have `overdue` around in my PATH to re-run to confirm the daemons aren't referencing deleted library file handles any more. Could your improved script consume `ls -d DEL` some way for one-off executions or is there a better way I'm not thinking of?